app/main.py

```diff
@@ -1,23 +1,36 @@
 import init_django_orm  # noqa: F401
 
-from django.db.models import QuerySet
+from django.db.models import QuerySet, Q, F
+from db.models import LoyaltyProgram, LoyaltyProgramParticipant, Customer
 
 
 def all_loyalty_program_names() -> QuerySet:
-    pass
+    return LoyaltyProgram.objects.all().values_list(
+        "name", "bonus_percentage"
+    )
 
 
 def not_active_customers() -> QuerySet:
-    pass
+    return LoyaltyProgramParticipant.objects.filter(
+        last_activity__year=2021
+    ).values("customer__first_name",)
 
 
 def most_active_customers() -> QuerySet:
-    pass
+    return LoyaltyProgramParticipant.objects.filter().values_list(
```

Comment:
Why do you need `.fitler()` - empty filter method?

Reply:
ouch, not needed, delete